portfolio/src/main/webapp/script.js

```diff
@@ -13,20 +13,28 @@
 // limitations under the License.
 
 /**
- * Show/hide the description inside the element which has title as id.
- */
-function toggleContent(sectionId) {
-  if (document.getElementById(sectionId) == null) {
-      return;
+* Insert/remove the class in the element.
+*/
+function toggleClass(element, classAttribute) {
+  if (!element || typeof element.className !== "string" || !classAttribute || typeof classAttribute !== "string") {
+    return;
```

Comment:
Nit: Consider if this should instead be throwing an exception. A caller might not notice that their input values are invalid if nothing happens.

Reply:
In this case if there is an error, I can't do anything to recover inside the first function, because it is caused by a wrong `sectionId`, so would be right to throw an exception and don't catch it?

Reply:
This depends heavily on the consumer/caller of the function. You want to e.g. avoid uncaught exceptions in your software but if all calls are directly controlled by you, it's a bug in your code if `toggleClass` is not called with valid arguments. My point here was that it might be hard to spot such a bug if the function just doesn't do anything vs if it throws an exception. In my opinion it would be ok to in this case throw an exception and not catch it so you would quickly notice that something is broken when trying it out.